backend/src/Controller/Admin/CourseCrudController.php

```diff
@@ -41,6 +41,9 @@ public function configureFields(string $pageName): iterable
         yield AssociationField::new('newCourses')
             ->autocomplete()
             ->setFormTypeOption('by_reference', false);
+        yield AssociationField::new('IdenticalCourses')
```

Comment:
Gives an error, because the field in Course.php is $identicalCourses, with a small i.
```suggestion
        yield AssociationField::new('identicalCourses')
```